### ISSUE TITLE
Fix position of header title.

### ISSUE THREAD

**Issue by paniash** [paniash/website#1]

**Comment by paniash** [comment]
As per a slack discussion (quite sometime back), could you help me out @tlienart?

**Comment by tlienart** [comment]
find the `@media (min-width: 940px)` in `basic.css` and set `width` to the same width as the `franklin-content` div with the same screen rule, so something like this

```css
@media (min-width:940px)
header { width: 830px; margin-left: auto; margin-right: auto; }
```

https://github.com/paniash/website/blob/a3fc265667681786532135cacc66c54411599d6e/_css/basic.css#L85

https://github.com/paniash/website/blob/a3fc265667681786532135cacc66c54411599d6e/_css/franklin.css#L109

**Comment by paniash** [comment]
@tlienart Thanks a lot! Fixed in [`9e9692a`](https://github.com/paniash/website/commit/9e9692afb29d2915be0385d59de57a3b4a540e28).
